velox/dwio/dwrf/common/Common.h

```diff
@@ -54,6 +59,7 @@ constexpr WriterVersion WriterVersion_CURRENT = WriterVersion::DWRF_7_0;
  */
 std::string writerVersionToString(WriterVersion kind);
 
+// Stream Kind of Dwrf
```

Comment:
Comment starts with capitalized letter and ends with `.`. Same for below.

Reply:
OK